<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>play_VScomputer</class>
 <widget class="QDialog" name="play_VScomputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0.0432692, y2:1, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>580</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0, y2:0.665, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>250</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>610</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0, y2:0.665, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));
font: 14pt &quot;Yu Gothic&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>250</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>610</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0, y2:0.665, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));
font: 14pt &quot;Yu Gothic&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>   computer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>220</y>
     <width>16</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0, y2:0.665, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>250</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>610</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>760</y>
     <width>1541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0.0432692, y2:1, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>430</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>430</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00961538, y2:0, stop:1 rgba(255, 255, 255, 255));
font: 72pt &quot;Showcard Gothic&quot;;
color: rgb(38, 48, 97);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>220</y>
     <width>16</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0, y2:0.665, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>471</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0242308, y1:0.079, x2:0, y2:0.665, stop:0.394231 rgba(70, 95, 177, 255), stop:1 rgba(62, 147, 238, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>90</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 28pt &quot;Simplified Arabic Fixed&quot;;</string>
   </property>
   <property name="text">
    <string>Computer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 28pt &quot;Simplified Arabic Fixed&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>60</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 48pt &quot;Stencil&quot;;</string>
   </property>
   <property name="text">
    <string> VS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Showcard Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Showcard Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_15</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>label_8</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>pushButton_14</zorder>
  <zorder>label_11</zorder>
  <zorder>pushButton_12</zorder>
  <zorder>pushButton_13</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_14</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_10</zorder>
  <zorder>label_9</zorder>
  <zorder>label_pic3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
